README.md

```diff
@@ -3,8 +3,8 @@
 ### Add NewRelic Server Monitoring
 
 1. SSH to one of the hosts
-2. git clone https://github.com/johanneswuerbach/newrelic-sysmond-service
-3. Edit `newrelic-sysmond.service` and replace `YOUR_NEW_RELIC_KEY` with our NewRelic Key and `CUSTOM_HOSTNAME=%H` with `CUSTOM_HOSTNAME=webprod-deis-%H`
+2. git clone https://github.com/lorieri/coreos-newrelic
+3. Edit `newrelic-sysmond.service` and replace `YOUR_NEW_RELIC_KEY` with our NewRelic Key and `-n %H` with `-n webprod-deis-%H`
```

Comment:
Why not copy the service file into this repo with a comment about the original source? Then we can apply tweaks like custom hostname prefixes directly.